.github/workflows/ubuntu_test.yml

```diff
@@ -28,6 +28,7 @@ jobs:
             ffmpeg \
             libavcodec-dev \
             libavformat-dev \
+            libavdevice-dev \
```

Comment:
actually, let's drop everything but libavdevice-dev here, too, so if it stops depping on everything necessary we'll detect it